packages/react/src/FilteredActionList/FilteredActionListWithModernActionList.tsx

```diff
@@ -104,20 +119,22 @@ export function FilteredActionList({
       },
     },
     [
-      // List ref isn't set while loading.  Need to re-bind focus zone when it changes
-      loading,
+      // List container isn't in the DOM while loading.  Need to re-bind focus zone when it changes.
+      listContainerElement,
     ],
   )
 
   useEffect(() => {
     // if items changed, we want to instantly move active descendant into view
     if (activeDescendantRef.current && scrollContainerRef.current) {
-      scrollIntoView(activeDescendantRef.current, scrollContainerRef.current, {...menuScrollMargins, behavior: 'auto'})
+      scrollIntoView(activeDescendantRef.current, scrollContainerRef.current, {
+        ...menuScrollMargins,
+        behavior: 'auto',
+      })
     }
   }, [items])
 
   useScrollFlash(scrollContainerRef)
-  useAnnouncements(items, listContainerRef, inputRef)
```

Comment:
could removing this have any impact on people using FilteredActionList directly without a SelectPanel or is that a non-issue?

Reply:
I was under the impression that `FilteredActionList` was only used internally since it's not documented on primer.style, etc. However I just did a [search](https://primer-query.githubapp.com/?name=%22FilteredActionList%22) for it and there is one usage in dotcom 🤦

Reply:
it's also in our Storybook https://primer.style/react/storybook/?path=/story/components-filteredactionlist--default